cityofaustin/atd-vz-data
# 8461 fatality people tab

## Associated issues
Closes cityofaustin/atd-data-tech/issues/8461

We have decided to nix the names and photos that were originally mocked into this issue due to concerns of not wanting PII in our database, so this PR just adds ethnicity and gender fields to the Peoples tab.

## Testing
**URL to test:** 
https://deploy-preview-1125--atd-vze-staging.netlify.app

**Steps to test:**

1. Go to a crash and navigate to the Related Records section.
2. Click on the Peoples tab, there should now be a Race/Ethnicity field and a Gender field for Primary and Other Persons tables.
3. Update their values and refresh the page, make sure they are still there.

Database changes associated with this PR:
Adding object relationships between the `prsn_ethnicity_id` and the `atd_txdot__ethnicity_lkp` table and between  `prsn_gndr_id` and the `atd_txdot__gndr_lkp` table. These relationships were added to both the `atd_txdot_person` table and the `atd_txdot_primaryperson` table.

---
#### Ship list
